extensions/amp-analytics/0.1/events.js

```diff
@@ -171,6 +186,14 @@ function isVideoTriggerType(triggerType) {
   return triggerType.startsWith('video');
 }
 
+/**
+ * @param {string} triggerType
+ * @return {boolean}
+ */
+function isCustomBrowserTriggerType(triggerType) {
+  return Object.values(BrowserEventType).indexOf(triggerType) > -1;
```

Comment:
`Object.values` is not support by IE. One option is to use `Object.keys` & then extract the values.